### PR TITLE
[Events] Added the three closest events

### PR DESCRIPTION
**Pull request by thomasbisignani**
Adds a three closest events module on the event page.

capture d ecran de 2018-09-10 15-20-23

### CODE REVIEW

File: front/style/pages/_committee.scss [hunk 1]
```diff
@@ -351,6 +351,42 @@
...
+            .card {
+                &:hover {
+                    h2 {
+                        color: $black;
+                    }
+                }
+                .box-header {
+                    &:hover {
+                       h2 {
+                        color: $black--half;
```
Comment (ottaviano) [inline front/style/pages/_committee.scss]: CS: indent 4 sps

File: templates/events/show.html.twig [hunk 2]
```diff
@@ -170,5 +170,40 @@
...
         </section>
+
+        {% if eventsNearby %}
+            <section class="committee-event-nearby l__wrapper--narrow">
+                <h2 class="text--large">{{ 'event.next_events_nearby'|transchoice(eventsNearby|length) }}</h2>
+                <ul class="l__row l__row--wrap l__row--top space--20-0 l__row--h-stretch-around l__mobile--col">
+                    {% for eventNearby in eventsNearby %}
+                        <li class="card card--content l__col--bottom">
+                            <div class="box-header">
+                                <a href="{{ path('app_event_show', { slug: eventNearby.slug}) }}" class="headline link--no-decor">
```
Comment (ottaviano) [inline templates/events/show.html.twig]: CS: 
`eventNearby.slug}` -> `eventNearby.slug }`

File: templates/events/show.html.twig [hunk 3]
```diff
@@ -170,5 +170,40 @@
...
+                                    <h4 class="text--summary text--gray b__nudge--bottom-medium"><i class="fa fa-clock-o" aria-hidden="true"></i> Le {{ eventNearby.beginAt|localizeddate('medium', 'none')|capitalize }}, {{ event.beginAt|date('G\\hi') }}</h4>
+                                    <img class="search-map"
+                                         src="{{ path('map_url', { latitude: eventNearby.latitude, longitude: eventNearby.longitude }) }}"
+                                         title="Carte d'accès à l'événement {{ eventNearby.name }}"
+                                         alt="{{ eventNearby.inlineFormattedAddress }}"
+                                    />
+                                </a>
+                            </div>
+                            <div class="box-bottom">
+                                <h4 class="text--summary b__nudge--bottom-tiny"><i class="fa fa-map-marker" aria-hidden="true"></i> {{ eventNearby.cityName}}</h4>
```
Comment (ottaviano) [inline templates/events/show.html.twig]: CS: `eventNearby.cityName}}` -> `eventNearby.cityName }}`

File: templates/events/show.html.twig [hunk 4]
```diff
@@ -170,5 +170,40 @@
...
+                                <div class="share">
+                                    <span role="button" onclick="App.share('facebook', '{{ url('app_event_show', { slug: eventNearby.slug}) }}', '{{ eventNearby.name|escape('js') }}')">
```
Comment (ottaviano) [inline templates/events/show.html.twig]: CS: `slug}` -> `slug }`

File: templates/events/show.html.twig [hunk 5]
```diff
@@ -170,5 +170,40 @@
...
+                                        <i class="fa fa-facebook-square"></i>
+                                    </span>
+                                        <span role="button" onclick="App.share('twitter', '{{ url('app_event_show', { slug: eventNearby.slug}) }}', '{{ eventNearby.name|escape('js') }}')">
```
Comment (ottaviano) [inline templates/events/show.html.twig]: CS: same :)

Comment (ottaviano) [inline templates/events/show.html.twig]: CS: too many spaces before `<span>`

File: templates/events/show.html.twig [hunk 6]
```diff
@@ -170,5 +170,40 @@
...
+                                    />
+                                </a>
+                            </div>
+                            <div class="box-bottom">
+                                <h4 class="text--summary b__nudge--bottom-tiny"><i class="fa fa-map-marker" aria-hidden="true"></i> {{ eventNearby.cityName }}</h4>
+                                <div class="share">
+                                    <span role="button" onclick="App.share('facebook', '{{ url('app_event_show', { slug: eventNearby.slug }) }}', '{{ eventNearby.name|escape('js') }}')">
+                                        <i class="fa fa-facebook-square"></i>
+                                    </span>
+                                        <span role="button" onclick="App.share('twitter', '{{ url('app_event_show', { slug: eventNearby.slug} ) }}', '{{ eventNearby.name|escape('js') }}')">
```
Comment (Remg) [inline templates/events/show.html.twig]: `{ slug: eventNearby.slug}` => `{ slug: eventNearby.slug }`
:trollface:

**Pull request merged**